package.json

```diff
@@ -103,7 +103,8 @@
     "did-jwt-vc": "3.1.3",
     "ethr-did": "2.3.9",
     "jsonld": "npm:@digitalcredentials/jsonld@^5.2.1",
-    "isomorphic-webcrypto": "link:./node_modules/@sphereon/isomorphic-webcrypto"
+    "isomorphic-webcrypto": "link:./node_modules/@sphereon/isomorphic-webcrypto",
+    "@sphereon/musap-react-native": "^0.0.1-unstable.129"
```

Comment:
Why are we not using explicit versions when targeting our instable releases? Do not use ^ or ~ in these cases!

Reply:
switched to explicit version in #40